solutions/telemetry/console/producer/.gitignore

```diff
@@ -0,0 +1,43 @@
+#Maven
```

Comment:
I think that keeping the `.gitignore` at the root of the project or app will make the trick for us and avoid repeating the file.

Reply:
Oh, just an old habit, I like to have the .gitignore file in any individual project. But I can create a single one of course